<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>881</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="sub_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>881</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QSlider" name="figure_Slider">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>19</width>
       <height>121</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 1, 255);
background-color: rgb(224, 255, 108);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Exit"/>
    <addaction name="action_Print"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menu_Music">
    <property name="title">
     <string>&amp;Music</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Zoom">
    <property name="title">
     <string>&amp;Zoom</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>&amp;Info</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Documentation"/>
   </widget>
   <widget class="QMenu" name="menu_Database">
    <property name="title">
     <string>&amp;Database</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Script">
    <property name="title">
     <string>&amp;Script</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Advanced">
    <property name="title">
     <string>&amp;Advanced</string>
    </property>
    <addaction name="action_Properties"/>
    <addaction name="action_Image"/>
    <addaction name="action_ImageWindow"/>
    <addaction name="action_Config"/>
    <addaction name="action_SetLayout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Music"/>
   <addaction name="menu_Zoom"/>
   <addaction name="menu_Database"/>
   <addaction name="menu_Script"/>
   <addaction name="menu_Advanced"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/fileopen.png</normaloff>:/images/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/filesave.png</normaloff>:/images/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/filesaveas.png</normaloff>:/images/images/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Documentation">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/blender.png</normaloff>:/images/images/blender.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/x.png</normaloff>:/images/images/x.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/vnc.png</normaloff>:/images/images/vnc.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/gv.png</normaloff>:/images/images/gv.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/editmirror.png</normaloff>:/images/images/editmirror.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_Properties">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/editmirrorhoriz.png</normaloff>:/images/images/editmirrorhoriz.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/clock.png</normaloff>:/images/images/clock.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="action_Image">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/emacs.png</normaloff>:/images/images/emacs.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/back.png</normaloff>:/images/images/back.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="MenuResources.qrc">
     <normaloff>:/images/images/usb.png</normaloff>:/images/images/usb.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_ImageWindow">
   <property name="text">
    <string>ImageWindow</string>
   </property>
  </action>
  <action name="action_Config">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="action_SetLayout">
   <property name="text">
    <string>setLayout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MenuResources.qrc"/>
 </resources>
 <connections/>
</ui>
